Jetcaster/app/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt

```diff
@@ -128,10 +130,27 @@ fun PlayerContent(
         // As the Player UI content changes considerably when the device is in tabletop posture,
         // we split the different UIs in different composables. For simpler UIs that don't change
         // much, prefer one composable that makes decisions based on the mode instead.
-        if (devicePosture is DevicePosture.TableTopPosture) {
-            PlayerContentTableTop(uiState, devicePosture, onBackPress)
-        } else {
-            PlayerContentRegular(uiState, onBackPress)
+        when (devicePosture) {
+            is DevicePosture.TableTopPosture ->
+                PlayerContentTableTop(uiState, devicePosture, onBackPress)
+            is DevicePosture.BookPosture ->
+                PlayerContentBook(uiState, devicePosture, onBackPress)
+            is DevicePosture.Separating ->
```

Comment:
Hi I was planning to use DevicePosture in Reply demo as well. I was wondering if we should have a standard device posture class for compose covering all cases or these are the most of the possible states? cc @alexvanyo

Reply:
At the moment these are the possible states but there may be foldable devices in the future with more than one `FoldingFeature` that may offer additional postures.

Reply:
Thanks for letting me know 👍🏽